mycrt-backend/src/server.py

```diff
@@ -107,21 +108,64 @@ def get_rds_instances():
 
     return jsonify({'status': response['ResponseMetaData']['HTTPStatusCode'], 'error': response['Error']['Code']})
 
-@app.route('/login-test', methods=['PUT'])
+@login_manager.request_loader
+def load_user_from_request(request):
+    auth = request.authorization
+    if auth:
+        username = auth.username
+        password = auth.password
+        user = user_datastore.find_user(username=username)
+        if user is not None and verify_password(password, user.password):
+            return user
+    return None
+
+@app.route('/login', methods=['POST'])
 def login():
-    user_repository.register_user(username="test-user", password="test123", email='test@test.com', access_key="test_access_key", secret_key="test_secret_key")
-
+    user = current_user
+    if user is not None:
+        login_user(user)
+        return jsonify({"status" : "success",

```

Comment:
successful status responses are sent as 200

Reply:
So this section doesn't actually return anything since it redirects back to '/'. I removed this.